packages/p2p/src/pages/my-profile/payment-methods/add-payment-method/add-payment-method-form.jsx

```diff
@@ -118,6 +118,7 @@ const AddPaymentMethodForm = ({ should_show_separated_footer = false }) => {
                                                 onChange={handleChange}
                                                 name={payment_method_field[0]}
                                                 required={!!payment_method_field[1].required}
+                                                value={field.value || ''}
```

Comment:
is this necessary?

Reply:
yeah we need to pass value which was the cause of the console error since the input field starts as uncontrolled, meaning the value was undefined/not set, and then it suddenly switches to being a controlled input, where the Formik loads all the values. So this will make sure the input field is already controlled and has a value by default, before it gets loaded with the passed default values from Formik props